### PR TITLE
Added Oracle Database Book

### PR DESCRIPTION
**Pull request by CodeSudo**
added Oracle database book

## What does this PR do?
Add resource

### Description
Oracle database book notes for professionals compiled by stackoverflow
### How do we know it's really free?
goalkicker.com provides provides this free
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2442,6 +2442,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Developing Time-Oriented Database Applications in SQL](https://www2.cs.arizona.edu/~rts/tdbbook.pdf) - Richard T. Snodgrass (PDF)
 * [Essential SQL](https://www.programming-books.io/essential/sql) - Krzysztof Kowalczyk and  Stack Overflow Documentation project (HTML)
 * [Introduction to SQL](https://github.com/bobbyiliev/introduction-to-sql) - Bobby Iliev (Markdown, PDF)
+* [Oracle Database Book](https://goalkicker.com/OracleDatabaseBook/) Compiled from StackOverflow Documentation (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:2445]: does this link work for you? https://goalkicker.com/OracleDatabaseBook/OracleDatabaseNotesForProfessionals.pdf ?
Don't invent titles! Use the title given by the resource.

Comment (eshellman) [reply books/free-programming-books-langs.md]: you don't want to use the link direct to pdf? or did it redirect you?

Comment (eshellman) [reply books/free-programming-books-langs.md]: the website is a hot mess of ads; the pdf is fine. just one link please.

Comment (CodeSudo) [reply books/free-programming-books-langs.md]: ok then i will just keep the pdf file link.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**